### ISSUE TITLE
Inverted camera

### ISSUE THREAD

**Issue by rico345100** [googlevr/gvr-unity-sdk#1030]
### SPECIFIC ISSUE ENCOUNTERED
In Unity Editor seems fine, however run with my mobile device, whole world just flipped.

### HARDWARE/SOFTWARE VERSIONS
- Unity: 2019.1.0f2
- Google VR SDK for Unity: 1.2
- Device manufacturer, model, and O/S: Samsung Galaxy S7

### STEPS TO REPRODUCE THE ISSUE
 1. Add GvrPointerPhysicsRaycaster in Main Camera
 2. Add new empty gameobject as Main Camera and drag GvrReticlePointer into Main Camera
 3. Build and run

**Comment by rico345100** [comment]
It works with 2018.x versions, and only happens 2019.1.0f2. Is it GVR SDK issue or Unity issue?

**Comment by YugiyO** [comment]
I also against with this issue on Samsung device(Note 9), but on Lenovo Phab 2 it works fine.

**Comment by rico345100** [comment]
@YugiyO It's not clear solution, but try with Unity 2018.x works for me.

**Comment by Xuplus** [comment]
Same with me, tried with a OnePlus 3 and an LG g3, both the camera is inverted using Unity 2019.1.1f1 and 2019.1.0f2.
Glad to hear 2018.x is working properly

**Comment by markchy** [comment]
what 2018 version is it specifically? Im having the same problem here too. Thanks.

**Comment by Xuplus** [comment]
The last one available (2018.3.14f1) works fine with Google VR

**Comment by rico345100** [comment]
@markchy I tested with 2018.3.12f1. Maybe latest 2018.x will works too.

**Comment by christougher** [comment]
Looks like it works normally with OpenGles3 but doesn't work on Metal (IOS) or Vulkan (Android).  Unfortunately for me my project requires Metal for ARKit as well.

**Comment by adilicious5** [comment]
Any update on this issue? I'm facing the same problem on the latest beta release 2019.2.0b4.

**Comment by markchy** [comment]
Yes I got it to work, kind of. I used GVR SDK v1.0.0 on an android 4. It works normal on android 4 but inverted on android 6. I still dont know why it does that.

**Comment by christougher** [comment]
Unity's response to my bug report:

"We are aware that the issue only occurs with Metal and Vulcan.
At this moment it is verified that the issue is fixed in the latest ALPHA version of Unity 2019.3.0a3.
I can't guarantee that the fix is available to 2019.1 versions, but the backports are opened for the fix to be implemented to this version.
I can't tell when it will be backported or will it be backported."

**Comment by mariovski** [comment]
Sweet! I went to player settings and removed Vulcan Api and stayed with openGLES3.
It works fine now on 2019.1.2f1 and Android Huawai p20 lite and others.

**Comment by christougher** [comment]
For a workaround on IOS see the other issue thread: 
https://github.com/googlevr/gvr-unity-sdk/issues/1031#issuecomment-501907953

**Comment by christougher** [comment]
Fix is included in 2019.2.0f1
